### PR TITLE
Cart

### PR DESCRIPTION
**Pull request by eakansh125**

### CODE REVIEW

**Review by sri-rishi — CHANGES_REQUESTED** [review]
image
 
UI is not responsive

**Review by sri-rishi — COMMENTED** [review]
Just a suggestion next time reduce class name size like image can be img

**Comment by eakansh125** [comment]
> image
> 
> UI is not responsive

I will be making it responsive i the future, thank you

File: screens/cart.html [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+                            </form>
+                        
+                        </div>
+                    </div>
+                </div>
+            </div>
+
+
+
+
```
Comment (pradipgharge) [inline screens/cart.html]: Remove extra spacing here.

File: style.css [hunk 2]
```diff
@@ -231,7 +279,7 @@ body{
     box-shadow: 1px 1px 10px 1px gray;
 }
 
-.image>img{
+.image-vertical>img{
     width: 100%;
     height: 320px;
```
Comment (manish-GH) [inline style.css]: use rem here

File: style.css [hunk 3]
```diff
@@ -302,6 +350,18 @@ body{
...
+}
+
+.vertical{
+    display: flex;
+    flex-direction: column;
+}
+
 /* Card sizes */
 .card-medium{
     width: 300px;
```
Comment (manish-GH) [inline style.css]: use rem

**Review by malaykhakhar — APPROVED** [review]
The code looks good. No suggestions as of now.
UI also looks good with minimalistic design.
You can merge it.

File: style.css (hunk 3 above)
Comment (pradipgharge) [inline style.css]: use rems

**Review by manish-GH — APPROVED** [review]
Both the code and the UI look good. Remove the unnecessary spacings and replace px with rem wherever possible. Then you are good to merge

**Review by pradipgharge — APPROVED** [review]
A little suggestion on the UI, your primary and secondary buttons look the same on the card. Add some distinction (like a background color) to differentiate b/w both

**Comment by eakansh125** [comment]
> Both the code and the UI look good. Remove the unnecessary spacings and replace px with rem wherever possible. Then you are good to merge

Changing px to rems wherever possible

**Comment by eakansh125** [comment]
> A little suggestion on the UI, your primary and secondary buttons look the same on the card. Add some distinction (like a background color) to differentiate b/w both

Will surely take care of that in future. Removing extra spaces for now.

**Pull request merged**